### PR TITLE
fwupd: update to 1.4.1

### PR DESCRIPTION
**Pull request by marmeladema**
Current version of `fwupd` stopped working. It fails with an error:
```
Failed to update metadata for lvfs: failed to set protocol: Invalid crypto engine
```

Upgrading to last version seems to fix the issue.

### CODE REVIEW

File: common/shlibs [hunk 1]
```diff
@@ -3860,3 +3860,6 @@ libtagparser.so.9 tagparser-9.1.2_1
 libqtutilities.so.6 qtutilities-6.0.4_1
 libkdsoap.so.1.9.0 KDSoap-1.9.0_1
 libstoken.so.1 stoken-0.92_1
+libjcat.so.1 libjcat-0.1.2_1
+libfwupd.so.2 fwupd-1.4.1_1
```
Comment (abenson) [inline common/shlibs:3864]: These probably aren't needed since I think they're only used in the same package

File: srcpkgs/libjcat/template [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Template file for 'libjcat'
+pkgname=libjcat
+version=0.1.2
+revision=1
+build_style=meson
+build_helper="gir"
+hostmakedepends="gnutls-tools pkg-config vala"
+makedepends="json-glib-devel gnutls-devel gpgme-devel"
+short_desc="Library for reading and writing Jcat files"
+maintainer="Orphaned <orphan@voidlinux.org>"
```
Comment (abenson) [inline srcpkgs/libjcat/template:10]: Orphaning a package already?

File: common/shlibs (hunk 1 above)
Comment (marmeladema) [reply common/shlibs]: Removed.

File: srcpkgs/libjcat/template (hunk 2 above)
Comment (marmeladema) [reply srcpkgs/libjcat/template]: Fixed.

**Comment by abenson** [comment]
This needs to be rebased on master before it can be merged

**Comment by Gottox** [comment]
@marmeladema please rebase! :smile:

**Comment by marmeladema** [comment]
Done!

**Pull request merged**